tests/smoke-composer.yaml

```diff
@@ -26,9 +26,6 @@ input:
             repo: dependabot/smoke-tests
             directory: /composer
             commit: 509a98a45a6f933ffd49ca7f2c26815c693b3d76
-        credentials-metadata:
-            - host: github.com
-              type: git_source
```

Comment:
This removal was accidental, fixed in:
* https://github.com/dependabot/smoke-tests/pull/246